### PR TITLE
Export dos outcomes

### PR DESCRIPTION
**Pull request by robyoung**
Depends on #36 

[#113152753](https://www.pivotaltracker.com/story/show/113152753)
[#113152881](https://www.pivotaltracker.com/story/show/113152881)
[#113153283](https://www.pivotaltracker.com/story/show/113153283)

This is similar to the script for exporting DOS Lots but for one for Outcomes, one for Specialists and one for User Research Participants.

After discussion with @cathroony we have decided to go wide and expand the capabilities and locations into columns as this is easier to work with.

### CODE REVIEW

File: dmscripts/export_dos_suppliers.py [hunk 1]
```diff
@@ -42,16 +42,62 @@ def inner(record):
     return inner
 
 
-def add_services(client, framework_slug):
+def add_services(client, framework_slug, lot=None, status=None):
```
Comment (TheDoubleK) [inline dmscripts/export_dos_suppliers.py]: Feels like this is more `add_drafts` than `add_services`, but no biggy.

File: scripts/export-dos-outcomes.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+from docopt import docopt
+from dmscripts.env import get_api_endpoint_from_stage
+from dmscripts.export_dos_suppliers import (
+    find_services_by_lot, FRAMEWORK_SLUG, make_fields_from_content_questions, write_csv
+)
+from dmapiclient import DataAPIClient
+from dmutils.content_loader import ContentLoader
+
+
```
Comment (TheDoubleK) [inline scripts/export-dos-outcomes.py]: Is it worth configuring a logger for this script, the same as the others?

**Comment by TheDoubleK** [comment]
Sorry for delay in finishing the review of these - given they've been run already and the outputs seem good (as far as I know?) I'm pretty sure they're fine :+1:

**Comment by robyoung** [comment]
That thumbs up wave is the best thing I've seen today!

**Pull request merged**